tests/util_tests/test_shotgun_path.py

```diff
@@ -8,14 +8,13 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-import os
 import sys
-import copy
 
-from tank_test.tank_test_base import *
+from tank_test.tank_test_base import setUpModule, TankTestBase # noqa
```

Comment:
It tells PEP8 to not QA that line. It complains because we are importing setUpModule which looks like it isn't used. It actually is, but indirectly by the unit testing framework. Doing this gives us a clean bill of health for that file.